source/integrate/community-for-mattermost-for-microsoft-teams.rst

```diff
@@ -0,0 +1,71 @@
+Community for Mattermost for Microsoft 365
+==========================================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+Connect with the Community for Mattermost for seamless collaboration.
```

Comment:
This first sentence is a bit hard to parse. 

What's the #1 motivator to try this from the customer's perspective?

Reply:
@lieut-data - my latest commit proposes some simplification in places where I was struggling to parse meaning. If you're aligned, I can update the docs file/URL accordingly.